share/doc/CMakeLists.txt

```diff
@@ -0,0 +1,32 @@
+if(NOT VOTCA_SPHINX_DIR)
+  set(VOTCA_SPHINX_DIR "${CMAKE_CURRENT_BINARY_DIR}")
+endif()
+
+file(GLOB NOTEBOOKS "${PROJECT_SOURCE_DIR}/GROMACS/Methane/QMMM_GROMACS.ipynb")
+list(APPEND NOTEBOOKS "${PROJECT_SOURCE_DIR}/LAMMPS/Thiophene/QMMM_LAMMPS.ipynb")
+list(APPEND NOTEBOOKS "${PROJECT_SOURCE_DIR}/tools/dftgwbse_CH4/DFTGWBSE_ENERGY.ipynb")
+list(APPEND NOTEBOOKS "${PROJECT_SOURCE_DIR}/tools/dftgwbse_CO_geoopt/DFTGWBSE_OPTIMIZATION.ipynb")
+# list(APPEND NOTEBOOKS "${PROJECT_SOURCE_DIR}/GROMACS/KMC_Methane/GROMACS_KMC.ipynb")
+list(APPEND NOTEBOOKS "${PROJECT_SOURCE_DIR}/LAMMPS/KMC_Thiophene/LAMMPS_KMC.ipynb")
+
+find_program(JUPYTER_EXECUTABLE NAMES jupyter "Interactive computing environment (https://jupyter.org/)")
+find_program(PIP_EXECUTABLE NAMES pip "Python package installer (https://pip.pypa.io/en/stable/)")
+
+add_custom_target(jupyter_client COMMAND ${PIP_EXECUTABLE} install wheel ipykernel jupyter_client sphinx>=2.1 nbsphinx --user)
```

Comment:
I really don't like the idea that we are automatically installing some crap in the user home directory.

We should just check if the required python modules are there (see the check for cma in csg-tutorials) and then fail/ don't build anything if they are don't. We can leave the pip install as a custom command, but the user has to run it manually.

Reply:
I am going to remove the pip command for the moment being and I am going to follow your advice about checking the required python modules.